doc-content/enterprise-only/openshift/imagestreams-file-install-proc.adoc

```diff
@@ -1,64 +1,47 @@
 [id='imagestreams-file-install-proc']
 = Ensuring the availability of image streams 
 
-You must ensure that the image streams that are required for the deployment are available in your OpenShift environment. Some versions of the OpenShift environment include the necessary image streams. You must check if they are available. If they are not available, you must install the
-ifdef::DM[`rhdm70-image-streams.yaml`]
-ifdef::PAM[`rhpam70-image-streams.yaml`]
-file.
+You must ensure that the image streams that are required for the deployment are available in your OpenShift environment. Some versions of the OpenShift environment include the necessary image streams. You must check if they are available. If they are not available, you must install the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-image-streams.yaml` file.
 
 .Procedure
 . Run the following commands:
 +
 ifdef::DM[]
-[subs="verbatim,macros"]
+[subs="attributes,verbatim,macros"]
```

Comment:
Throughout the trial doc, the variables in the snippets are still not rendering in the output, and I think it may be because "attributes" needs to be "attributes+". If not that, then perhaps "verbatim" is conflicting with "attributes". You may need to toy around and see what you find. Let me know if you can't crack it and I'll have a closer look in your branch.

Reply:
No, it's because I did not reupload the trial doc after fixing this issue.